### PR TITLE
Cleanup board flashing pages

### PR DESCRIPTION
**Pull request by kprosise**
Style and grammar fixes across a number of pages. Markup had semantic newlines applied. Removed images already covered in the getting started page, such as the screenshots of the UI—this will make maintaining easier.

Switches/pins/hardware components are now in bold rather than inline or plain.

Ran vale linter within editor. Checked output in browser, which initially lead to a number of revisions. At this point all looks without issue, however there is a lot of content to go over multiple times.

This commit addresses to FFTK-2788

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

Cleanup!

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

## Comments

Not sure if there are any boards here which we want to remove. For the post part keeping this commit to cleanup, however I am aware we will likely want to do a bigger overhaul of this section at some point.

### CODE REVIEW

**Comment by angolini** [comment]
I'm ok with the PR, but I still want to see the final result (waiting for the ci link) to make sure the standard used with the pins are matching (I don't remember how I used it on the variables)

**Comment by kprosise** [comment]
> I'm ok with the PR, but I still want to see the final result (waiting for the ci link) to make sure the standard used with the pins are matching (I don't remember how I used it on the variables)

The way you did it was actually what inspired me to update all the others!

**Review by angolini — APPROVED** [review]
I don't know why the build fails, but the overall review is LGTM.
A well-needed cleanup!

**Comment by vanmaegima** [comment]
I'd like to review this one from the built pages, so I'll wait until the docs build passes.

**Comment by doanac** [comment]
Docs for f46a8af947e500db81f032aeb26cec98696cbe49 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2463/docs/artifacts/html/index.html

**Comment by kprosise** [comment]
@angolini @vanmaegima It has been built!

**Comment by angolini** [comment]
For me, it's a thumbs up LGTM

File: source/reference-manual/boards/am62xx-sk-prepare.rst [hunk 1]
```diff
@@ -1,17 +1,23 @@
 Building
 --------
 
-When building the Cortex R5 u-boot, it builds for all different variants:
+When building the Cortex R5 u-boot, all variants are built:
```
Comment (vanmaegima) [inline source/reference-manual/boards/am62xx-sk-prepare.rst:4]: We should have a linter for U-Boot (there's many variants through the docs)
```suggestion
When building the Cortex R5 U-Boot, all variants are built:
```

File: source/reference-manual/boards/am62xx-sk-prepare.rst [hunk 2]
```diff
@@ -20,49 +26,20 @@ The following changes the default to ``gp`` so the image produced boots on that
...
-
-#. Organize all the files like the tree below::
-
-      ├── lmp-factory-image-am62xx-evm.wic
-      └── ti-mfgtool-files-am62xx-evm
-          ├── flash.sh
-          ├── tiboot3.bin
-          ├── tispl.bin
-          └── uboot.img
-
```
Comment (vanmaegima) [inline source/reference-manual/boards/am62xx-sk-prepare.rst:65]: Any reason why we removed the mfgtool-files mention for TI boards?

File: source/reference-manual/boards/am62xx-sk-prepare.rst [hunk 3]
```diff
@@ -20,49 +26,20 @@ The following changes the default to ``gp`` so the image produced boots on that
...
+#. Download the necessary files from ``https://app.foundries.io/factories/<factory>/targets``:
 
-#. Click the latest Target with the :guilabel:`platform-devel` trigger.
+  a. Click the latest Target with the :guilabel:`platform` trigger.
 
-   .. figure:: /_static/boards/generic-steps-1.png
-      :align: center
-      :width: 300
+  b. Expand the **run** in the :guilabel:`Runs` section which corresponds with the name of the board.
+     **Download the Factory image for that machine.**
```
Comment (vanmaegima) [inline source/reference-manual/boards/am62xx-sk-prepare.rst:37]: Should we provide the actual name of the image here? We are doing it anyway on the next step.

File: source/reference-manual/boards/am62xx-sk-prepare.rst [hunk 4]
```diff
@@ -20,49 +26,20 @@ The following changes the default to ``gp`` so the image produced boots on that
...
-   with the name of the board and **download the Factory image for that
-   machine.**
-
-   | E.g: ``lmp-factory-image-am62xx-evm.wic.gz``
-
-   .. figure:: /_static/boards/am62xx-sk-steps-2.png
-      :align: center
-      :width: 600
-#. Extract the file ``lmp-factory-image-am62xx-evm.wic.gz``::
+#. Extract the file::
```
Comment (vanmaegima) [inline source/reference-manual/boards/am62xx-sk-prepare.rst:39]: The numbering here on the compiled page is wrong (shows as '1' as well).

File: source/reference-manual/boards/am64xx-sk-prepare.rst [hunk 5]
```diff
@@ -20,27 +26,17 @@ The following changes the default to ``gp`` so the image produced boots on that
...
 Download necessary files from ``https://app.foundries.io/factories/<factory>/targets``
 
-#. Click the latest Target with the :guilabel:`platform-devel` trigger.
+#. Click the latest Target with the :guilabel:`platform` trigger.
 
-   .. figure:: /_static/boards/generic-steps-1.png
-      :align: center
-      :width: 300
+#. Expand the **run** in the :guilabel:`Runs` section by clicking on the ``+`` sign corresponding with the name of the board.
+   **Download the Factory image for that machine.**
```
Comment (vanmaegima) [inline source/reference-manual/boards/am64xx-sk-prepare.rst:37]: Same here.

File: source/reference-manual/boards/secure-boot-note.rst [hunk 6]
```diff
@@ -1,7 +1,5 @@
...
-    The instructions in this section also applies to those boards with secure
-    boot enabled. There are references on how to perform common instructions
-    along with the flow.
+    The instructions in this section also applies to those boards with secure boot enabled.
+    There are references on how to perform common instructions along with the flow.
 
-    The :ref:`ref-security` Reference Manual details the required background
-    for secure boot.
\ No newline at end of file
+    :ref:`ref-security` details the required background for secure boot.
```
Comment (vanmaegima) [inline source/reference-manual/boards/secure-boot-note.rst:5]: It's better to point directly to https://ci.foundries.io/projects/fio-docs/builds/2463/docs/artifacts/html/reference-manual/security/security.html#secure-boot-hardware-root-of-trust

File: source/reference-manual/boards/apalis-imx6-prepare.rst [hunk 7]
```diff
@@ -1,46 +1,38 @@
...
-
-          .. figure:: /_static/boards/apalis-imx6-steps-2.png
-            :width: 769
-            :align: center
+        lmp-factory-image-<machine-name>.wic.gz
+        u-boot-<machine-name>.itb
+        sit-<machine-name>.bin
+        SPL-<machine-name>
 
 #. Extract the file ``lmp-factory-image-apalis-imx6.wic.gz``::
```
Comment (vanmaegima) [inline source/reference-manual/boards/apalis-imx6-prepare.rst:21]: @angolini Do we still need to extract the file for imx images?

File: source/reference-manual/boards/apalis-imx6-prepare.rst [hunk 8]
```diff
@@ -1,46 +1,38 @@
...
-#. Download and extract the file ``mfgtool-files-apalis-imx6.tar.gz``::
+   .. prompt::
 
-      tar -zxvf mfgtool-files-apalis-imx6.tar.gz
+         tar -zxvf mfgtool-files-apalis-imx6.tar.gz
 
-#. Organize all the files like the tree below::
+#. Organize the files as in the tree below::
 
       ├── lmp-factory-image-<machine-name>.wic.gz
```
Comment (vanmaegima) [inline source/reference-manual/boards/apalis-imx6-prepare.rst:37]:
```suggestion
      ├── lmp-factory-image-<machine-name>.wic.gz
```
We mention to extract the files in the last step however here we list the .wic.gz file. Needs confirmation from @angolini  if this should be extracted or not.

**Comment by vanmaegima** [comment]
Some things that I noticed:
`Preparation` text is not consistent across the pages (check beaglebone, apalis and am6x pages for example). I would prefer to keep the text consistent.
Also numbering is odd. Maybe we can reuse the same note for all pages if possible.
Need to confirm with Daiane about the extracted image, this will apply to all imx boards.

File: source/reference-manual/boards/am62xx-sk-prepare.rst (hunk 4 above)
Comment (kprosise) [reply source/reference-manual/boards/am62xx-sk-prepare.rst:39]: Blast, I thought I got that worked out. Thanks!

File: source/reference-manual/boards/apalis-imx6-prepare.rst (hunk 7 above)
Comment (angolini) [reply source/reference-manual/boards/apalis-imx6-prepare.rst:21]: for the nxp images I'm sure we don't
But I don't think I changed it for apalis, hold on

**Comment by angolini** [comment]
>`Preparation` text is not consistent across the pages (check beaglebone, apalis and am6x pages for example). I would prefer to keep the text consistent.

It's indeed not consistent, but I'm not sure how much we can make them equal, as this is the place where the architecture will differ. But I understand at least the leading text could be the same (am I right in my interpretation?)

And we also can think about how the preparation is different between "normal" and "closed" board.

File: source/reference-manual/boards/apalis-imx6-prepare.rst (hunk 8 above)
Comment (vanmaegima) [reply source/reference-manual/boards/apalis-imx6-prepare.rst:37]: I think we don't need to worry too much about apalis, it's not a lmp reference anymore.

**Comment by vanmaegima** [comment]
As discussed with @kprosise in the call today: many of my reports are not critical changes (like making the pages consistent). We can fix it later and just handle the critical stuff (like compressed images or mfgtool comments).

File: source/reference-manual/boards/am62xx-sk-prepare.rst (hunk 2 above)
Comment (kprosise) [reply source/reference-manual/boards/am62xx-sk-prepare.rst:65]: Honestly I have no idea. I remember cleaning it up, not removing it. I will put it back in.

File: source/reference-manual/boards/am62xx-sk-prepare.rst (hunk 3 above)
Comment (kprosise) [reply source/reference-manual/boards/am62xx-sk-prepare.rst:37]: Could the name vary, such as if a Fast First Target image is used?

Comment (vanmaegima) [reply source/reference-manual/boards/am62xx-sk-prepare.rst:37]: Yes, for FFT - lmp-base-console-image, for normal builds - lmp-factory-image.
File `generic-prepare.rst` seems to cover both names.

Comment (angolini) [reply source/reference-manual/boards/am62xx-sk-prepare.rst:37]: I still don't believe the FFT image name is different...........

Comment (kprosise) [reply source/reference-manual/boards/am62xx-sk-prepare.rst:37]: Yes, I recall updating the generic-prepare, seemed to have skipped doing it here. I'll check all the prepare files.

**Comment by doanac** [comment]
Docs for 2e1d6781ef4072e7cdbe9febace17c8d7fa06da8 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2476/docs/artifacts/html/index.html

File: source/reference-manual/boards/apalis-imx6-prepare.rst (hunk 8 above)
Comment (kprosise) [reply source/reference-manual/boards/apalis-imx6-prepare.rst:37]: does this need to be changed?

Comment (vanmaegima) [reply source/reference-manual/boards/apalis-imx6-prepare.rst:37]: No, we can keep it as it is (I believe apalis is not covered by FFT).

File: source/reference-manual/boards/apalis-imx6-prepare.rst (hunk 7 above)
Comment (vanmaegima) [reply source/reference-manual/boards/apalis-imx6-prepare.rst:21]: @kprosise This needs to be changed for all i.mx but apalis-imx6/8
Now we don't need the extract step

**Comment by doanac** [comment]
Docs for 2fd9fbb874c4fa4c02d81aacf67990d5656ea236 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2482/docs/artifacts/html/index.html

**Pull request merged**